Update remove-former-employee-step-3.md (#7591)
MicrosoftDocs/microsoft-365-docs: microsoft-365/admin/add-users/remove-former-employee-step-4.md

- TRN_M365B
- OKR_SMB_Videos
- AdminSurgePortfolio
- m365solution-removeemployee
search.appverid:
- BCS160
- MET150
- MOE150
description: "Follow these steps to forward a former employee's email to another employee or convert to a shared mailbox."
---

# Step 4 - Forward a former employee's email to another employee or convert to a shared mailbox

In this step, you assign the former employee's email address to another employee, or convert the user's mailbox to a shared mailbox.

## Convert former employee's mailbox to a shared mailbox

> When you convert a user's mailbox to a shared mailbox, all of the existing email and calendar is retained. Only now it's in a shared mailbox where several people will be able to access it instead of one person. You can convert a shared mailbox back to a user (private) mailbox at a later date if you want.

Reviewer: ```suggestion
When you convert a user's mailbox to a shared mailbox, all of the existing email and calendar are retained. Only now it's in a shared mailbox where several people will be able to access it instead of one person. You can convert a shared mailbox back to a user (private) mailbox at a later date if you want.
```

> - Creating a shared mailbox is the less expensive way to go because you won't have to pay for a license **as long as the mailbox is smaller than 50GB**. Over 50GB and you'll need to assign a license to it.

Reviewer: ```suggestion
- Creating a shared mailbox is the less expensive way to go because you won't have to pay for a license **as long as the mailbox is smaller than 50 GB**. If it is over 50 GB, you'll need to assign a license to it.
```

- If you convert the mailbox to a shared mailbox, all the old email will be available, too. This can take up a lot of space.

> - If you set up email forwarding, only *new* emails sent to the former employee will now be sent to the current employee.

Reviewer: ```suggestion
- If you set up email forwarding, only *new* emails sent to the former employee will be sent to the current employee.
```

> Follow these steps on how to [convert the user's mailbox to a shared mailbox](../email/convert-user-mailbox-to-shared-mailbox.md).

Reviewer: ```suggestion
Follow these steps to [convert the user's mailbox to a shared mailbox](../email/convert-user-mailbox-to-shared-mailbox.md).
```

## Forward a former employee's email to another employee

 > [!IMPORTANT]

>  > If you're setting up email forwarding or a shared mailbox, at the end, don't delete the former employee's account. The account needs to be there to anchor the email forwarding or shared mailbox.

Reviewer: ```suggestion
 > If you're setting up email forwarding or a shared mailbox, in the end, don't delete the former employee's account. The account needs to be there to anchor the email forwarding or shared mailbox.
```
